src/PFXImportPowershell/PFXImportPS.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Do we need to update the pre-reqs in the readme?  It currently says VS 15+. What version is v4.7.2 in?